subsys/bluetooth/mesh/Kconfig

```diff
@@ -289,6 +289,19 @@ config BT_MESH_CRPL
 	  protection list. This option is similar to the network message
 	  cache size, but has a different purpose.
 
+choice BT_MESH_RPL_STORAGE_MODE
```

Comment:
Why is there only one choice?

Reply:
We need the choice when we are going to add BT_MESH_RPL_STORAGE_MODE_EMDS in nrf. Because the Kconfig system will merge all the choices that are made into one choice even if they are defined different places.

Reply:
Originally it was two, but @PavelVPV wanted to optimize the `No Persistent` usecase should be covered by settings. Like @Balaklaka said it has to be a choice to allow additional entries to be added in downstream

Reply:
Hm. We need to suggest the universal approach that is possible to use with emds.
As a third-party developer, I do not understand what I need to choose here for my own implementation of rpl storage.